### DISCUSSION TITLE
如何在theme.ts中locales.footer的自定义html中获取到$isDarkMode？（万分感谢回答，在线等）

### DISCUSSION THREAD

**Discussion by heshimang** [vuepress-theme-hope/vuepress-theme-hope#2308] [Q&A] [answered]
如何在theme.ts中locales.footer的自定义html中获取到$isDarkMode？我需要在这里拿到主题并改变底部的图片
![image](https://user-images.githubusercontent.com/19547994/196321306-946a2586-8bd2-45fc-9e36-4f4b6633f449.png)

**Comment by Mister-Hope** [comment, marked as answer]
你不能，建议写两张图片，并用css选择器控制显示其中之一。

你可以参考 imageMark 是如何实现的并进行同样的操作

**Reply by heshimang** [reply]
收到，刚试了一下，是可以的，很感谢。

**Reply by Mister-Hope** [reply]
（LZ信了你的在线等还特意在路上回的）

**Comment by Mister-Hope** [comment]
更新，如果你需要定制化主题的任何一个部分，你永远拥有一个强大的最后手段，也就是自己写一个满足需求的组件进行替换，主题文档的教程高级部分有提到相关操作。

我们假定你在页脚中放置的图片非常大，以至于你不想接受同时加载两张图片并用css隐藏一个，那你可以直接替换PageFooter，并在组件中通过 useDarkmode 拿到 isDarkmode 进行条件渲染。 这还能在一定程度上增强性能（因为页脚不再需要动态注入html而是一个完全的定制组件）

**Reply by heshimang** [reply]
赞👍🏻
